data/ecosystems/g/gravita-protocol.toml

```diff
@@ -0,0 +1,31 @@
+# Ecosystem Level Information
+title = "Gravita Protocol"
```

Comment:
could you please connect this ecosystem to Ethereum and Arbitrum?

Reply:
Do you mean to add these repos to the Ethereum and Arbitrum toml?

Reply:
Hey @eherrerosj Have added the ecosystems

Reply:
oh, actually Gravita should be a subecosystem of Ethereum and Arbitrum and not the other way around. In other words, the way you are requesting the changes is Gravita is a parent of Ethereum and Arbitrum in the taxonomy